src/utils.ts

```diff
@@ -56,6 +70,29 @@ export function formatCustomerName(customer: Customer): string {
   return `${lastName}, ${firstName}`;
 }
 
+export function formatRegistrationNumbers(permits: Permit[]): string {
+  const registrationNumbers: string[] = [];
+  // eslint-disable-next-line
+  permits.forEach(function (permit) {
+    registrationNumbers.push(permit.vehicle.registrationNumber);
+  });
+  return registrationNumbers.join(', ');
```

Comment:
I think this is a legit complaint from the eslint. The `map` function would be a better option. You can do something like:

```suggestion
 return permits.map(permit => permit.vehicle.registerationNumber).join(', ')
```